rfcs/protocols/template-protocol.md

```diff
@@ -113,23 +113,23 @@ By convention, state names use lower-kebab-case but are compared
 case-insensitively and ignoring punctuation.
 
 State management in protocols is a deep topic. For more information, please
-see [State Details and State Machines](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0003-protocols/state-details.md).
+see [State Details and State Machines](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0003-protocols/state-details.md).
```

Comment:
This link is broken. Use https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0003-protocols#state-details-and-state-machines instead.